### DISCUSSION TITLE
Error: Does anyone encounter this error "The request has been black-holed" ???

### DISCUSSION THREAD

**Discussion by KumaKode** [Floorp-Projects/Floorp#977] [Q&A]
I was browsing this website:

![image](https://github.com/Floorp-Projects/Floorp/assets/116707143/d6787333-20ad-456e-bc4c-8d2a698b4fbd)

and got the following error. I tried on diff browser and error didn't pop-up. can anyone tell me the solution?

Also the hcaptcha is working very slower on this browser or doesn't work.

**Comment by CutterKnife** [comment]
It looks like the same thing as the so-called "404 Not Found".

**Reply by regularentropy** [reply]
True. It's 404

**Reply by KumaKode** [reply]
yes but when i open the link in other browser, this error won't show, i simply get my result.

**Comment by regularentropy** [comment]
Can't confirm. In my case, —works on both Floorp and Vivaldi.
> yes but when i open the link in other browser, this error won't show, i simply get my result.
